backend-api/app/routers/stream.py

```
@@ -37,15 +38,14 @@ async def reader():
                     continue
                 data = msg.get("data")
-                # Per-message trace is extremely noisy (ffmpeg progress lines every ~100ms).
-                if logger.isEnabledFor(logging.DEBUG):
-                    preview = data[:120] if isinstance(data, str) else data
-                    logger.debug("[SSE %s] redis: %s", task_id[:8], preview)
+                logger.info(f"[SSE {task_id[:8]}] Received Redis message: {data[:100] if isinstance(data, str) else data}")
+                sys.stdout.flush()
                 await queue.put(str(data))
         except asyncio.CancelledError:
             pass
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)